test/cublas.jl

```diff
@@ -298,6 +298,67 @@ end
         dhA = CuArray(hA)
         x = rand(elty,m)
         dx = CuArray(x)
+
+        function pack(A, uplo)
+            AP = Vector{elty}(undef, (n*(n+1))>>1)
+            k = 1
+            for j in 1:n
+                for i in (uplo==:L ? (j:n) : (1:j))
+                    AP[k] = A[i,j]
+                    k += 1
+                end
+            end
+            return AP
+        end
+
+        if elty in ["Float32", "Float64"]
+            # pack matrices
+            sAPU = pack(sA, :U)
+            dsAPU = CuVector(sAPU)
+            sAPL = pack(sA, :L)
+            dsAPL = CuVector(sAPL)
+
+            @testset "spmv!" begin
+                # generate vectors
+                y = rand(elty,m)
+                # copy to device
+                dy = CuArray(y)
+                # execute on host
+                BLAS.spmv!('U',alpha,sAPU,x,beta,y)
+                # execute on device
+                CUBLAS.spmv!('U',alpha,dsAPU,dx,beta,dy)
+                # compare results
+                hy = Array(dy)
+                @test y ≈ hy
+                # execute on host
+                BLAS.spmv!('U',alpha,sAPL,x,beta,y)
+                # execute on device
+                CUBLAS.spmv!('U',alpha,dsAPL,dx,beta,dy)
+                # compare results
+                hy = Array(dy)
+                @test y ≈ hy
+            end
+
+            @testset "spr!" begin
+                # uncomment lines with code below when julia 1.8 is released
```

Comment:
Already enable this test then, but in a VERSION branch. You can look up the exact version number you need by passing the merge commit sha of your PR to the commit-name.sh script in julia/contrib.

Reply:
thanks for the tip!  i didn't know how to get the version needed.